feature-1/one-more-file.txt

```diff
@@ -0,0 +1 @@
+This file was also added in feature-1 branch
```

Comment:
Its really good that you have decided to add this line of code

Reply:
thanks buddy